integration/apps/hpcg/download_and_build.sh

```diff
@@ -31,6 +31,7 @@
 #
 
 set -e
+set -x
```

Comment:
I'm not opposed to this change, but set -x is not present in the other download and build scripts.  Maybe it should be.